operations/flags.go

```diff
@@ -105,6 +106,13 @@ func addTasksFlag(flags ...cli.Flag) []cli.Flag {
 	})
 }
 
+func addParametersFlag(flags ...cli.Flag) []cli.Flag {
+	return append(flags, cli.StringSliceFlag{
+		Name:  parametersFlagName,
+		Usage: "specify parameters as comma-separated KEY=VALUE pairs",
```

Comment:
I'm not sure that we want to have this be comma-delimited. The library allows you to do something like 
`john@johns-MacBook-Pro evergreen % clients/darwin_amd64/evergreen patch -p mci --parameters a=b --parameters r=t
Set [{a b} {r t}] as the default parameters for project 'mci'? no 
`
which I think is cleaner. Also recommending that people separate these with commas means that you can't have a comma in a parameter value

Reply:
that's reasonable to me